pkg/config/config.go

```diff
@@ -180,12 +189,12 @@ func NewMicroshiftConfig() *MicroshiftConfig {
 		SubjectAltNames: subjectAltNames,
 		NodeName:        nodeName,
 		NodeIP:          nodeIP,
+		BaseDomain:      "microshift.example.com",
```

Comment:
Shouldn't this be "example.com", as "microshift" is also the clusterName?
(i.e. we'd end up getting`*.apps.microshift.microshift.example.com`)

Reply:
+1 true, at least until we resolve/allow adding a cluster name